Library/Formula/httpfs2.rb

```diff
@@ -0,0 +1,25 @@
+class Httpfs2 < Formula
+  desc "FUSE filesystem for mounting files from http servers"
+  homepage "http://httpfs.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/httpfs/httpfs2/httpfs2-0.1.5.tar.gz"
+  sha256 "01cb4bb38deb344f540da6f1464dc7edbdeb51213ad810b8c9c282c1e17e0fc1"
+
+  depends_on "pkg-config" => :build
+  depends_on "dpkg" => :build
+  depends_on "asciidoc" => :build
+  depends_on "docbook-xsl" => :build
+  depends_on "gnutls"
+  depends_on :osxfuse
```

Comment:
If this absolutely needs a mandatory `osxfuse` dependency it needs to be submitted to https://github.com/Homebrew/homebrew-fuse.